File: js/BeginnersSelection/02_PlacingMarbles.js
// https://atcoder.jp/contests/abs/tasks/abc081_a

Object.prototype.let = function(f) {
  return f(this.valueOf());
}

require('fs')
  .readFileSync('/dev/stdin', 'utf8')
  .split('')
  .map(x => +x)
  .reduce((a, x) => a + x)
  .let(console.log);

  // https://atcoder.jp/contests/abs/submissions/21478010
  

File: py/BeginnersSelection/02_PlacingMarbles.py
# https://atcoder.jp/contests/abs/tasks/abc081_a

class W:
  def __init__(self, v): self.v = v
  let = lambda self, f: W(f(self.v))
  map = lambda self, f: W(map(f, self.v))

W(input()) \
  .map(int) \
  .let(sum) \
  .let(print)

# https://atcoder.jp/contests/abs/submissions/21602387
